compiler-core/src/metadata/tests.rs

```diff
@@ -444,7 +444,7 @@ fn module_fn_value_regression() {
                         start: 52,
                         end: 1100,
                     },
-                    supported_targets: SupportedTargets::all(),
```

Comment:
This change and the following one was just to have some more variety in the tests

Reply:
I'd prefer new tests for these, to make it clear that it's deliberate with each test only testing one thing.